kubeflow-pipelines.yaml

```diff
@@ -12,23 +12,23 @@ package:
 environment:
   contents:
     packages:
-      - wolfi-base
+      - argo-cd
+      - build-base
       - busybox
       - ca-certificates-bundle
-      - build-base
+      - eslint
+      - git
       - go
       - go-licenses
-      - git
-      - argo-cd
-      - python3-dev
-      - py3-setuptools
-      - py3-gpep517
-      - py3-wheel
-      - py3-pip
+      - jq
       - kubectl
       - nodejs
-      - jq
-      - eslint
+      - py3-gpep517
+      - py3-pip
+      - py3-setuptools
+      - py3-wheel
+      - python3-dev
+      - wolfi-base
```

Comment:
Does this need to include apk? 🤔

Reply:
@developer-guy need to check this comment here, if that is not needed we can resolve this for now

Reply:
I don't think that we need this one, let me remove it.